infra/scripts/test_ubuntu_runtime_mixed.sh

```diff
@@ -55,8 +55,8 @@ echo "GeneratedTests.squeeze_relaxed" >> $SKIPLIST_PREFIX.union
 
 # Run the test
 export OP_BACKEND_Conv2D="cpu"
-export OP_BACKEND_MaxPool2D="acl_cl"
-export OP_BACKEND_AvgPool2D="acl_neon"
+export OP_BACKEND_Pool2D="acl_cl"
+export OP_BACKEND_FullyConnected="acl_neon"
```

Comment:
@hseok-oh Is this line related to `MaxPool` and `AvgPool` changes?

Reply:
@glistening Right. #3853 merges internal IR `MaxPool2D` and `AvgPool2D` to `Pool2D`, and OP-backend mapping uses internal IR list. So these env variables cannot handle backend mapping.

Reply:
@hseok-oh I am asking line 59 (`OP_BACKEND_FullyConnected`). ( I chosen line 59, but github highlights including 58. )

Reply:
`OP_BACKEND_MaxPool2D` and `OP_BACKEND_AvgPool2D` do nothing. So remain OP code on inception test model is `FullyConnected`, `Concat`, and `Softmax`. So I choosed `FullyConnected`.